source/MainPanel.cpp

```diff
@@ -124,6 +124,9 @@ void MainPanel::Step()
 
 	engine.Step(isActive);
 
+	if(isActive)
+		player.StepMissionTimers(GetUI());
```

Comment:
We don't want to step timers if the engine is paused. 
```suggestion
	if(isActive && !engine.IsPaused())
		player.StepMissionTimers(GetUI());
```